### PR TITLE
Update README.md for guidance, starting somewhere

### PR DESCRIPTION
**Pull request by barriebyron**
start a draft README to provide guidance, per https://github.com/cosmos/advocates/issues/15

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,29 @@
...
+- version
+- screen shots
+- other relevant information
+
+## Issue triage 
+
+As we scale our content teams, issue triage is a fluid process that we're still working out. Thank you for your patience. Thank you for caring.
+
+- For Cosmos SDK and Starport docs, tag @barriebyron 
+- For Cosmos SDK tutorials, tag @toschdev
```
Comment (hxrts) [inline README.md:25]: some things not covered that should have owners:
- IBC protocol website https://ibcprotocol.org/
- rust IBC & hermes relayer (should be andy)
- typescript relayer
- tendermint core
- tendermint spec repo
- IBC spec repo
- cosmjs

**Comment by hxrts** [comment]
ty @barriebyron !

**Comment by barriebyron** [comment]
I don't know who and what to add here, but Elias from B9 can take it from here.

**Comment by hxrts** [comment]
btw, I don't have perms to merge, but this LGTM to start

**Review by hxrts — APPROVED** [review]

**Comment by barriebyron** [comment]
@okwme I would tag Elias for his review and attention if I knew his GitHub username. Can you tag him here?

**Comment by soniasingla** [comment]
@barriebyron is it ready to merge? :)

**Comment by barriebyron** [comment]
@soniasingla yes, ready, but we require 2 approvals in this repo. @8bitpal can approve?

**Review by toschdev — APPROVED** [review]

**Comment by barriebyron** [comment]
maybe @hxrts can approve so we can merge?

**Review by soniasingla — APPROVED** [review]

**Pull request merged**